.github/actions/setup-sql-server/Initialize-SqlServer.ps1

```diff
@@ -1,18 +1,20 @@
 if ($Env:RUNNER_OS -eq 'Windows')
 {
-    choco install sql-server-express --no-progress
+    choco install sql-server-2022 --no-progress
 }
 else
 {
+    $sqlServerName = 'sql2022'
+    $sqlServerVersion = '2022-latest'
```

Comment:
You can use only the version eg `$sqlServerVersion = '2022'` and with string interpolation, you can use it everywhere `"mcr.microsoft.com/mssql/server:${sqlServerVersion}-latest"`.

Why don't you want to create a script parameter for this?

Reply:
We are making changes because there is a bug in our script and we found a maintainability problem while fixing it.

Script parameter:
```
param (
    [string]$SqlServerVersion = '2022'
)

...
```

Variable:
```
...

$sqlServerVersion = '2022'

...
```

Reply:
I did change to param now.